webrtc/README.md

```diff
@@ -161,6 +165,51 @@ MUST pass an empty string. When receiving an `RTCDataChannel` via
 an empty string. This allows future versions of this specification to make use
 of the `RTCDataChannel` `label` property.
 
+## Closing an `RTCDataChannel`
+
+Some WebRTC implementations do not guarantee that any queued messages will be
+sent after a datachannel is closed.  Other implementations maintain separate
+outgoing message and transport queues, the status of which may not be visible
+to the user. Consequently we must add an additional layer of signaling to ensure
+reliable data delivery.
+
+When a node wishes to close a stream for writing, it SHOULD send a message with
+the `FIN` flag set, then proceed to wait for a `FIN_ACK` message from the remote
+node.
+
+If a `FIN` flag is received the node SHOULD respond with a `FIN_ACK`.
```

Comment:
If we flip the order of these two, it is easier to understand why we wait for a `FIN_ACK` after sending `FIN`.

Reply:
I've split the paragraph up and changed the order, hopefully the sequence of events is clearer now?